lm75bd/lm75bd.c

```diff
@@ -26,7 +26,28 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
+
+  uint8_t temp1 = temp_addr;
+  i2cSendTo(devAddr, &temp1, 1);
+
+  if (temp == NULL) {
+    return ERR_CODE_INVALID_ARG; 
+  }
```

Comment:
this should be checked at the beginning of the function since if the temp is invalid then we shouldn't be setting the pointer register